tasks/gulp/__tests__/after-build-dist.test.mjs

```diff
@@ -73,6 +77,10 @@ describe('dist/', () => {
       expect(javascript).toBeTruthy()
     })
 
+    it('should contain correct version number', () => {
+      expect(javascript).toContain(`t.version="${pkg.version}",`)
```

Comment:
It's likely to change infrequently

@querkmachine and I decided to keep it in but shall we drop the `t`?

```suggestion
      expect(javascript).toContain(`.version="${pkg.version}",`)
```

Reply:
I think we should drop the t, yeah. I fear this might be one of those things that just changes some day in the future and someone ends up wasting hours working out why. Best to avoid that scenario if we can do so easily.

Reply:
✅ Updated to ~`t.version`~ `.version`